packages/jest-core/src/TestScheduler.ts

```diff
@@ -201,6 +201,27 @@ class TestScheduler {
           aggregatedResults.snapshot.filesRemoved)
       );
     };
+    const buildExecError = (err: unknown): SerializableError => {
+      const fromString = (errString: string): SerializableError => {
+        const {message, stack} = separateMessageFromStack(errString);
+        if (stack.length > 0) {
+          return {message, stack};
+        }
+        const error = new ErrorWithStack(message, buildExecError);
+        return {message, stack: error.stack || ''};
+      };
```

Comment:
can we define this function at the top level rather than inline here?

Reply:
sure